engine/runtime/src/test/resources/Cycle_Test/src/Main.enso

```diff
@@ -1,3 +1,4 @@
-import Cycle_Test.Sub.Imp
+import project.Sub.Imp
```

Comment:
Can we _still_ import by the project name itself?

Reply:
I would think so, because the desugar process converts to exactly that, so there should be no reason why that would be disallowed.

Reply:
Project name + name space. So the original import here is invalid, but you could write `import local.Cycle_Test.Sub.Imp`